pyanaconda/ui/gui/spokes/lib/resize.py

```diff
@@ -100,129 +106,46 @@ def __init__(self, data, storage, payload):
         self._delete_button = self.builder.get_object("deleteButton")
         self._resize_slider = self.builder.get_object("resizeSlider")
 
-    def _get_description(self, part):
-        # First, try to find the partition in some known Root.  If we find
-        # it, return the mountpoint as the description.
-        for root in self.storage.roots:
-            for (mount, device) in root.mounts.items():
-                if device == part:
-                    return "%s (%s)" % (mount, root.name)
+    def _is_protected(self, device_name):
+        return device_name in self._protected_devices
+
+    def _get_partition_description(self, device_data, format_data):
+        # First, try to find the partition in some known root.
+        # If we find it, return the mount point as the description.
+        for root in self._roots:
+            for mount_point, device_name in root.mount_points.items():
+                if device_name == device_data.name:
+                    return "{mount_point} ({os_name})".format(
+                        mount_point=mount_point, os_name=root.os_name
+                    )
 
         # Otherwise, fall back on increasingly vague information.
-        if not part.isleaf:
-            return part.children[0].name
-        if getattr(part.format, "label", None):
-            return part.format.label
-        elif getattr(part.format, "name", None):
```

Comment:
Just making sure this is left out intentionally.

Reply:
The format name is provided by `format_data.description` below.